kubemarine/cri/__init__.py

```diff
@@ -32,6 +33,14 @@ def enrich_inventory(inventory: dict, cluster: KubernetesCluster) -> dict:
         if value in cluster.raw_inventory.get('services', {}).get('cri', {}):
             raise Exception(f"{key} is not used, please remove {value} config from `services.cri` section")
 
+    # Enrich containerdConfig
+    if cri_impl == "containerd":
+        containerd_config = cluster.inventory["services"]["cri"]['containerdConfig']
+        runc_options_path = 'plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options'
+        if not isinstance(containerd_config[runc_options_path]['SystemdCgroup'], bool):
+            containerd_config[runc_options_path]['SystemdCgroup'] = \
+                bool(strtobool(containerd_config[runc_options_path]['SystemdCgroup']))
```

Comment:
If `migrate_cri`, it is anyway necessary to convert `SystemdCgroup`, but it will be skipped in this case. Also I found similar part of code in `upgrade.upgrade_containerd`.